### ISSUE TITLE
MacOS: Move focus to tags doesn't work

### ISSUE THREAD

**Issue by Robertght** [Automattic/simplenote-electron#777]
#### Steps to reproduce
1. Navigate to any note or press any tag then press on  Control-tab to  Move focus to tags
2. Nothing happens.

#### What I expected
To switch the focus across tags

#### What happened instead
Nothing 

#### OS version
MacOS
Simplenote version 1.3.5

1239842-zen

**Comment by mirka** [comment]
I believe this is a report for simplenote-macos, not simplenote-electron.

Closing as a duplicate of https://github.com/Automattic/simplenote-macos/issues/227
